kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -2,6 +2,7 @@
 
   <core-base>
     <main-nav slot="nav"></main-nav>
+    <toolbar slot="above"></toolbar>
     <div slot="above" class="top-wrapper">
       <search-button class='search-btn'></search-button>
       <label for="chan-select" class="visuallyhidden">Filter User Type</label>
```

Comment:
Label is for "Switch Channels":
`<label for="chan-select" class="visuallyhidden">Switch Channels</label>`